### PR TITLE
Add mission statement

### PR DESCRIPTION
**Pull request by cagallo**
Is this a fix or a feature?
- [ ] Bug fix
- [X] Feature
- [ ] Refactor
 
Describe the feature - fix - refactor:

Added mission statement to HomePage component. 

How was this tested?
- [X] open index.html
- [ ] localhost
- [ ] terminal

Where should the reviewer start - test:

- go to our localhost:3000, view mission statement on home page
- also fixed integration test with wrong file path

Additional comments:

Feel free to add or edit anything you feel could be changed, I'm not married to anything and won't be offended!

### CODE REVIEW

File: src/Components/HomePage/HomePage.js [hunk 1]
```diff
@@ -6,9 +6,11 @@ const HomePage = () => {
     <section className='main-container columns'>
       <aside className='mission'>
         <h2>Our Mission</h2>
-        <p>To build connections between parents of baby’s in need and generous donors.</p>
-        <p>Additional messaging about our mission... </p>
-        <p>Message about providing resources and support..... Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation.</p>
+        <p>Our orgnanization strives to build connections between parents of babies in need and generous donors.</p>
```
Comment (janitastic) [inline src/Components/HomePage/HomePage.js:9]: spell check organization

File: src/Components/HomePage/HomePage.js [hunk 2]
```diff
@@ -6,9 +6,11 @@ const HomePage = () => {
...
+        <p>Human milk is the natural food for babies and provides them with optimal nutrition that allows them to reach their health potential.
+           We are here to help mothers who are not able to nurse their babies or produce enough milk by connecting them with mothers who produce more milk than their babies need. 
+           Together, we will help infants thrive by making donated breastmilk safe and accessible.</p>
+        <p>We understand that breastfeeding issues can lead to many different stressers and emotions. We are here for you and you are not alone. Please visit our Support page to view more resources.</p>
```
Comment (janitastic) [inline src/Components/HomePage/HomePage.js:13]: We should Link the support page here.

File: src/Components/HomePage/HomePage.js [hunk 3]
```diff
@@ -6,9 +6,11 @@ const HomePage = () => {
     <section className='main-container columns'>
       <aside className='mission'>
         <h2>Our Mission</h2>
-        <p>To build connections between parents of baby’s in need and generous donors.</p>
-        <p>Additional messaging about our mission... </p>
-        <p>Message about providing resources and support..... Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation.</p>
+        <p>Our orgnanization strives to build connections between parents of babies in need and generous donors.</p>
+        <p>Human milk is the natural food for babies and provides them with optimal nutrition that allows them to reach their health potential.
+           We are here to help mothers who are not able to nurse their babies or produce enough milk by connecting them with mothers who produce more milk than their babies need. 
+           Together, we will help infants thrive by making donated breastmilk safe and accessible.</p>
```
Comment (janitastic) [inline src/Components/HomePage/HomePage.js:12]: let's remove the words 'safe and' from this line.

File: src/Components/NeedMilk/NeedMilk.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import './NeedMilk.scss';
 import DonorCard from '../DonorCard/DonorCard';
-import {useEffect, useState} from 'react'
-import {useQuery, gql} from '@apollo/client'
+import {useEffect, useState} from 'react';
+import {useQuery, gql} from '@apollo/client';
```
Comment (janitastic) [inline src/Components/NeedMilk/NeedMilk.js:4]: great job on this fix!

**Review by janitastic — CHANGES_REQUESTED** [review]

File: src/Components/HomePage/HomePage.js (hunk 2 above)
Comment (devonwigle) [inline src/Components/HomePage/HomePage.js:13]: stressers -> stressors

Replace first sentence with the following:
The period of time after a baby is born is naturally a very stressful time. Emotions and pressure is high. We understand that breastfeeding issues can be a major contributor to a mother's stress level.

**Review by janitastic — CHANGES_REQUESTED** [review]
Let's double check the 'exact to' errors in the console after we merge this.

**Pull request merged**